docs/library/axi_adc_trigger/index.rst

```diff
@@ -201,5 +201,5 @@ Register Map
 References
 --------------------------------------------------------------------------------
 
+* :dokuwiki:`AXI ADC TRIGGER on wiki <resources/fpga/docs/axi_adc_trigger>`
 * :git-hdl:`library/axi_adc_trigger`
```

Comment:
This applies to all files:

No need to add a link to the wiki documentation, which is already ported (this document).
Also, please add for all IPs this text in front of the library links:

```suggestion
* HDL IP core at :git-hdl:`library/axi_adc_trigger`
```